pom.xml

```diff
@@ -621,6 +621,9 @@
         </profile>
         <profile>
             <id>kafka-0.10-dev</id>
+            <activation>
```

Comment:
Then we should do this version upgrade first, because 0.8 apis do not support extraction of timestamp. I will raise separate PR for activation of 0.10 profile before timestamp changes are merged.